sdmetrics/__init__.py

```diff
@@ -144,7 +144,7 @@ def _find_addons():
         try:
             addon = entry_point.load()
         except Exception:  # pylint: disable=broad-exception-caught
-            msg = f'Failed to load "{entry_point.name}" from "{entry_point.module_name}".'
+            msg = f'Failed to load "{entry_point.name}" from "{entry_point.value}".'
```

Comment:
`module_name` is only an attribute when `entry_point` is imported from `pkg_resources`. Since I changed it to import from `importlib` instead, that attribute doesn't exist anymore.